lightningd/opening_control.c

```diff
@@ -1192,7 +1252,7 @@ static const struct json_command fundchannel_complete_command = {
     "channels",
     json_fundchannel_complete,
     "Complete channel establishment with peer {id} for funding transaction"
-    "with {txid}. Returns true on success, false otherwise."
+    "with {psbt}. Returns true on success, false otherwise."
```

Comment:
I wonder if this will actually make us incompatible with LN Pool? surely not... as the pool probably produces PSBTs for peers to sign for the collectively built opening tx?

Reply:
We can always re-add a txid arg, but I really think that it's a great sanity-check!